storey/table.py

```diff
@@ -852,7 +861,11 @@ def get_features(self, timestamp, windows=None):
         # In case our pre aggregates already have the answer
         for win in windows:
             result[f"{self.name}_{self.aggregation}_{win[1]}"] = self._current_aggregate_values[win].value
-
+            if (
+                self.aggregation != "count"
+                and (count_features.get(f"{self.name}_count_{win[1]}", 0) if count_features else 1) == 0
```

Comment:
Isn't that more simply
```suggestion
                and count_features
                and count_features.get(f"{self.name}_count_{win[1]}", 0) == 0
```